source/features/follow-file-renames.tsx

```diff
@@ -22,7 +22,8 @@ async function findRename(lastCommitOnPage: string): Promise<File[]> {
 function init(): false | void {
 	const disabledPagination = select.all('.paginate-container [disabled], .paginate-container .disabled');
 	const url = new GitHubURL(location.href);
-
+	// Clear the search from the url, so it does not get passed to the rename link.
+	url.search = '';
```

Comment:
Good point, but then again this should be part of `GitHubURL` and be more specific to `path[]`, probably.

Reply:
Also make sure to add tests

Reply:
>  and be more specific to path[], probably.
I just noticed this. You mean to say only clear the path ones?

Reply:
Not sure, but the Older link doesn't work on https://github.com/yakov116/refined-github/commits/f23b687b3b89aa95a76193722cdfeff740646670/source/features/release-download-count.tsx?after=f23b687b3b89aa95a76193722cdfeff740646670+34

Reply:
Apparently that also happens without this PR. I'll open a new issue.

Reply:
Nope. It works correctly. The build wasn't updating 👌